### PR TITLE
Set grey background if the field is readonly

### PR DESCRIPTION
**Pull request by kosiakkatrina**
image

### CODE REVIEW

File: app/helpers/question_attribute_helper.rb [hunk 1]
```diff
@@ -10,6 +10,7 @@ def stimulus_html_attributes(question)
 private
 
   def numeric_question_html_attributes(question)
+    return { "style": "background-color: #f3f2f1;" } if question.read_only?
```
Comment (baarkerlounger) [inline app/helpers/question_attribute_helper.rb:13]: Might be better to add a CSS class? (Just a nit though)

**Review by baarkerlounger — APPROVED** [review]

File: app/helpers/question_attribute_helper.rb (hunk 1 above)
Comment (kosiakkatrina) [reply app/helpers/question_attribute_helper.rb:13]: I thought that, but it's just one thing. What are the benefits?

Comment (baarkerlounger) [reply app/helpers/question_attribute_helper.rb:13]: Not huge so I think either way is ok. Benefits are consistent place you know to look for styling and colours (it's probably where I'd check first), and quicker to reuse if we did have a second thing at some point?).

Comment (kosiakkatrina) [reply app/helpers/question_attribute_helper.rb:13]: I can't see us using it somewhere else right now. I'll just merge it in as is

**Pull request merged**